app/Http/Controllers/Auth/LoginController.php

```diff
@@ -16,16 +16,22 @@ public function signin(Signin $request)
     {
         $credentials = $request->only('email', 'password');
         try {
-            if (! $token = JWTAuth::attempt($credentials)) {
+            if (!$token = JWTAuth::attempt($credentials)) {
                 return response()->exception('Invalid credentials', 401);
             } else {
                 $userId = Auth::user()->id;
+                $user = User::find($userId);
+                $patients = $user->patients()
+                    ->select(['patient_id','first_name','last_name'])
+                    ->get()->toArray();
+
                 return response()->success([
                     'id' => $userId,
-                    'token' => $token
+                    'token' => $token,
+                    'patients' => $patients
```

Comment:
Shouldn't we pass this the same way we pass other arrays?
`->get()->values->all()`
And even after that, I think the mobile app uses models that parse the response with camelCased properties.
e.g. patientId instead of patient_id
This is why we needed transformers in other controllers